test/test.js

```diff
@@ -96,6 +97,25 @@ test('plan assertions with support for promises', function (t) {
 	});
 });
 
```

Comment:
Yeah, but that should be done in a separate PR.

- test.js
- promise.js
- observable.js